index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" href="style.css">
+	<title>MY PORTFOLIO</title>
+</head>
+<body>
+	<section class="hero-section">
+		<nav class="nav-bar">
+			<h1 class="logo">
+				My Logo
+			</h1>
+			<img src="./assets/images/Union.svg" alt="" srcset="" class="mobile-menu">
+		</nav>
+		<main class="headline bg-img">
+			<h1 class="greeting">
+				I’m Valentine Glad to see you!
+			</h1>
+			<p class="personal-details">
+				I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+			</p>
+			<h3 class="connect">
+				LET’S CONNECT
+			</h3>
+			<div class="social-links">
+				<a href="https://twitter.com/blaze_valentine"><img src="./assets/images/twitter_icon.svg" alt="" srcset=""></a>
+				<a href="https://www.linkedin.com/in/valentine-blaze"><img src="./assets/images/Linkedin_icon.svg" alt="" srcset=""></a>
+				<a href="http://medium.com"><img src="./assets/images/medium_icon.svg" alt="" srcset=""></a>
+				<a href="https://github.com/V-Blaze"><img src="./assets/images/github_icon.svg" alt="" srcset=""></a>
+				<a href="http://"><img src="./assets/images/Vector.svg" alt="" srcset=""></a>
+			</div>
```

Comment:
- [ ] Kindly consider wrapping all the social links inside a `ul`  and not a `div`. and each one of them inside a `li` for semantic purpose
- [ ] Kindly make sure you add the values for your alternative text `alt` attribute. As you can see their utility [here](https://www.w3schools.com/tags/att_area_alt.asp#:~:text=Definition%20and%20Usage,user%20uses%20a%20screen%20reader).